aea/cli/generate_key.py

```diff
@@ -57,14 +57,13 @@ def _generate_private_key(type_: str, file: Optional[str] = None) -> None:
     """
     if type_ == "all" and file is not None:
         raise click.ClickException("Type all cannot be used in combination with file.")
-    elif type_ == "all":
-        types = list(IDENTIFIER_TO_KEY_FILES.keys())
-    else:
-        types = [type_]
-    for type_ in types:
-        private_key_file = IDENTIFIER_TO_KEY_FILES[type_] if file is None else file
+    types = list(crypto_registry.supported_ids) if type_ == "all" else [type_]
+    for type__ in types:
+        private_key_file = (
+            PRIVATE_KEY_PATH_SCHEMA.format(type__) if file is None else file
+        )
         if _can_write(private_key_file):
-            create_private_key(type_, private_key_file)
+            create_private_key(type__, private_key_file)
```

Comment:
why double underscore?

Reply:
`type_` probably  can be named `type_option` or `type_selection`

Reply:
you reuse a variable from outer scope..it really doesn't matter here; just to avoid same var name